lib/Controller/RoomController.php

```diff
@@ -1898,6 +1910,15 @@ public function leaveRoom(string $token): DataResponse {
 
 		try {
 			$room = $this->manager->getRoomForUserByToken($token, $this->userId, $sessionId);
+
+			// Check if the user is banned from the room
+			try {
```

Comment:
Huhu? Banned users are (should be) removed by the moderators action. they will not be able to call this endpoint?